.golangci.yaml

```diff
@@ -105,11 +104,130 @@ linters-settings:
     custom-order: true
     skip-generated: true
   importas:
-    no-unaliased: true
+    no-unaliased: false
     no-extra-aliases: true
+    ## kubernetes APIs get the prefix `k`. As an example check `kcorev1`
+
     alias:
       - pkg: k8s.io/api/core/v1
         alias: kcorev1
+      - pkg: k8s.io/api/apps/v1
+        alias: kappsv1
+      - pkg: k8s.io/api/rbac/v1
+        alias: krbacv1
+      - pkg: k8s.io/apimachinery/pkg/runtime/schema
+        alias: kschema
+      - pkg: k8s.io/apimachinery/pkg/labels
+        alias: klabels
+      - pkg: k8s.io/apimachinery/pkg/runtime
+        alias: kruntime
+      - pkg: k8s.io/apimachinery/pkg/types
+        alias: ktypes
+      - pkg: k8s.io/apimachinery/pkg/apis/meta/v1/unstructured
+        alias: kunstructured
+      - pkg: k8s.io/apimachinery/pkg/apis/meta/v1
+        alias: kmetav1
+      - pkg: github.com/kyma-project/eventing-manager/api/eventing/v1alpha1
+        alias: eventingv1alpha1
+      - pkg: github.com/kyma-project/eventing-manager/api/eventing/v1alpha2
+        alias: eventingv1alpha2
+      - pkg: github.com/kyma-project/eventing-manager/api/operator/v1alpha1
+        alias: operatorv1alpha1
+      - pkg: github.com/kyma-incubator/api-gateway/api/v1beta1
+        alias: apigatewayv1beta1
+      - pkg: k8s.io/client-go/dynamic/fake
+        alias: kdynamicfake
+      - pkg: k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/fake
+        alias: kapiclientsetfake
+      - pkg: k8s.io/apimachinery/pkg/api/errors
+        alias: kerrors
+      - pkg: k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1
+        alias: kapiextensionsv1
+      - pkg: github.com/kyma-project/eventing-manager/testing
+        alias: eventingtesting
+      - pkg: k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset
+        alias: kapiclientset
+      - pkg: sigs.k8s.io/controller-runtime
+        alias: ctrl
```

Comment:
How about the alias `kctrl` or `skctrl`? For `sigs.k8s`

Reply:
i personally would prefer `kctrl`.

Reply:
`kctrl` sounds good